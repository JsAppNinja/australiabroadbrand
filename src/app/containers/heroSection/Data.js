export default {
  herotitle: "Share the love with Australia Broadband",
  heroDescription: "Earn 100$ for each friend that you bring on board",
  buttonContent: "Check address",
  cards: [
    {
      title: "",
      avatar: "",
      price: "",
      priceInfo: [],
      description: ""
    },
    {
      title: "",
      avatar: "",
      price: "",
      priceInfo: [],
      description: ""
    },
    {
      title: "",
      avatar: "",
      price: "",
      priceInfo: [],
      description: ""
    }
  ]
};
