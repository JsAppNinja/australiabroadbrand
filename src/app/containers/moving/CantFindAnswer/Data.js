export default [
  {
    desc: [
      'The connection time frame will depend on whether or not your new address has been connected to the nbn™ previously.',
      'If your new home has been previously connected to the nbn™, we’ll normally activate your Australia Broadband nbn™ service typically within 2-5 days. This may take longer if the former tenant or owner did not disconnect their nbn™ service before you moved in. ',
      'If it is the first time your new home is connecting to the nbn™ network, we’ll arrange the first available appointment with NBN Co for an NBN Co technician to connect your nbn™ service. Typically, new nbn™ connections are completed by the NBN Co within 5-20 business days. This varies depending on your nbn™ connection type as well as the availability of the NBN Co technicians in your area.',
    ],
  },
  {
    desc:
      'If I sign up now, will my nbn™ service be activated by the time I move in?',
  },
];
