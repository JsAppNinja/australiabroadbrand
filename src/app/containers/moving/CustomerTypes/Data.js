import NewCustomer from '../../../assets/new-customer.jpg';
import ExistingCustomer from '../../../assets/existing-customer.jpg';

export default {
  newCustomer: {
    type: 'new-customer',
    text: "I'm a new customer",
    img: NewCustomer,
  },
  existingCustomer: {
    type: 'existing-customer',
    text: "I'm an existing customer",
    img: ExistingCustomer,
  },
};
