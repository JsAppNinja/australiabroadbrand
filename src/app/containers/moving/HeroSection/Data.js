import MovingHeroLeft from '../../../assets/moving-hero-left.png';
import MovingHeroRight from '../../../assets/moving-hero-right.png';

export default {
  herotitle: 'Hassle free nbn™ moving service',
  heroDescription:
    'If you’re moving into a new place and need to connect to the nbn™, Australia Broadband provides an easy to switch & easy to connect service with no lock in contracts and no worries.',
  buttonContent: 'Check address',
  hero_left: MovingHeroLeft,
  hero_right: MovingHeroRight,
};
