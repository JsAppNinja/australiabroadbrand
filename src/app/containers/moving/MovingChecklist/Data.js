export default {
  checkpoints: [
    {
      desc:
        "if you're renting,complete the property's condition report-it's a good idea to take photos and email then to your landloard or property manger",
    },
    {
      desc:
        'make sure all your utilities are connected(including your belong broadband) and the hot water service is on',
    },
    {
      desc:
        "if you're moving some of your possessions out of storage interstate or overseas,check your inventory-contact the moving company innediately if anything is missing",
    },
  ],
};
