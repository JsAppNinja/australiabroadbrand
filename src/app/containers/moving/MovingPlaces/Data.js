export default [
  {
    mainQuestion: 'How do I move my Australia Broadband service with me?',
    mainAnswer:
      'You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.',
    questions: [
      {
        question: 'My new address is nbn™ ready. What do I do?',
        answer: 'That’s great! Simply follow the steps above',
      },
      {
        question: 'What if my new address is not nbn™ ready?',
        answer:
          'If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.',
      },
    ],
  },
  {
    mainQuestion:
      'What does it cost to move my Australia Broadband service to a new address?',
    mainAnswer:
      'We don’t charge you anything to move your Australia Broadband service to a new address.',
    questions: [
      {
        question:
          'Does Australia Broadband charge any relocation fees to connect nbn™ at my new address?',
        answer:
          'We don’t charge anything to move your Australia Broadband service to your new address. If a standard NBN Co installation is needed at your new address, it will also be completed at no cost to you.',
      },
      {
        question:
          'Does the NBN Co charge any connection or installation fees to connect my new address?',
        answer:
          'A small number of homes may be subject to an NBN Co Non-Standard Installation or a NBN Co New Development Charge of $300. If your new address is subject to any of these NBN Co charges, we’ll let you know before the installation and get your permission before you’re charged.',
      },
    ],
  },
  {
    mainQuestion:
      'If I sign up now, will my nbn™ service be activated by the time I move in?',
    mainAnswer:
      'The connection time frame will depends whether or not your new address has been connected to the nbn™ previously.',
    questions: [
      {
        question: 'My new home has previously been connected to the nbn™',
        answer:
          'If your new home has been previously connected to the nbn™, we’ll normally activate your Australia Broadband nbn™ service typically within 2-5 days. This may take longer if the former tenant or owner did not disconnect their nbn™ service before you moved in.',
      },
      {
        question:
          'It is the first time my new home is connecting to the to the nbn™',
        answer:
          'If it is the first time your new home is connecting to the nbn network, we’ll arrange the first available appointment with NBN Co for an NBN Co technician to connect your nbn™ service. Typically, new nbn™ connections are completed by the NBN Co within 5-20 business days. This varies depending on your nbn™ connection type as well as the availability of the NBN Co technicians in your area. We will let you know if you need an appointment after we have submited your order to the NBN Co. ',
      },
    ],
  },
  {
    mainQuestion:
      'What happens to my Australia Broadband VoIP number when I move?',
    mainAnswer:
      'When you move home you can take your Australia Broadband VoIP number with you, if you are moving within the same state. If you relocate to another state you will need a new number.',
  },
  {
    mainQuestion:
      'There is already a service connected at my new address but it’s not in my name, what can I do?',
    mainAnswer:
      'If the previous occupants have not disconnected their service, we’ll need to get some paperwork from you, before the existing service can be legally disconnected. Once the paperwork’s received and verified, you should be connected within 10 business days depending the type of nbn™ connection.',
  },
];
