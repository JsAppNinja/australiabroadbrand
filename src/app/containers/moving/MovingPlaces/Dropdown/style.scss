.dropdown {
  width: 800px;
  &:not(:last-child) {
    margin-bottom: 15px;
  }
  &__button {
    display: flex;
    justify-content: space-between;
    align-items: center;
    border-radius: 7px;
    border: 1px solid #67cebb;
    border-left: 3px solid #67cebb;
    position: relative;
    cursor: pointer;
    padding: 25px;
    font-weight: bold;
    text-align: left;
    &:after {
      content: '';
      background: #67cebb;
      height: 80%;
      width: 6px;
      border-radius: 15px;
      position: absolute;
      left: 0;
      top: 50%;
      transform: translate(-50%, -50%);
    }
    &__main-question {
      margin: 0;
    }
    &__circle {
      width: 30px;
      height: 30px;
      background: #d3f2ed;
      border-radius: 50%;
    }
  }
  &__content {
    p {
      font-family: Roboto;
      font-size: 14px;
      font-weight: normal;
      font-style: normal;
      font-stretch: normal;
      line-height: 1.71;
      letter-spacing: normal;
      color: #313541;
      padding: 0 25px 0 25px;
      text-align: justify;
      &:last-child {
        margin-bottom: 0;
      }
    }
    .question {
      font-weight: bold;
    }
  }
  .svg-inline--fa {
    color: #67cebb;
    width: 25px;
    height: 25px;
    display: flex;
    text-align: center;
    margin: 2px auto;
  }
  .separated-paragraph {
    margin-top: 10px;
    padding: 0;
  }
}
