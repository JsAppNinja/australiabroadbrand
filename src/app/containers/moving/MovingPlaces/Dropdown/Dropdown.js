import React, { Component, Fragment } from 'react';
import PropTypes from 'prop-types';
import { Arrow } from '../../../../components/Arrow';
import AnimateHeight from 'react-animate-height';

import './style.scss';

class Dropdown extends Component {
  render() {
    const { mainQuestion, mainAnswer, questions } = this.props.data.item;
    const { itemIdx, openedDropdownIdx } = this.props.data;
    const isCurrentOpened = openedDropdownIdx === itemIdx;
    return (
      <div className="dropdown">
        <div
          className="dropdown__button"
          onClick={() => this.props.expandDropdown(itemIdx)}
        >
          <p className="dropdown__button__main-question">{mainQuestion}</p>
          <div className="dropdown__button__circle">
            <Arrow rotate={isCurrentOpened ? true : false} />
          </div>
        </div>
        <AnimateHeight duration={500} height={isCurrentOpened ? 'auto' : 0}>
          <section className="dropdown__content">
            <p>{mainAnswer}</p>
            {questions
              ? questions.map((item, index) => (
                  <Fragment key={index}>
                    <p>{item.question}</p>
                    <p>{item.answer}</p>
                  </Fragment>
                ))
              : null}
          </section>
        </AnimateHeight>
      </div>
    );
  }
}

Dropdown.propTypes = {
  data: PropTypes.object.isRequired,
  expandDropdown: PropTypes.func.isRequired,
};

export default Dropdown;
