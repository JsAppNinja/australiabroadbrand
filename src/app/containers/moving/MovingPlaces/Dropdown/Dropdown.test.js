import React from 'react';
import { shallow, mount } from 'enzyme';

import Dropdown from './Dropdown.js';

import Data from '../Data';

const defaultProps = {
  openedDropdownIdx: null,
  expandDropdown: jest.fn(),
  data: {
    item: Data[0],
  },
};

describe('Dropdown tests', () => {
  const tree = shallow(<Dropdown {...defaultProps} />);
  const spyOnExpandDropdown = jest.spyOn(defaultProps, 'expandDropdown');
  it('should render dropdown section', () => {
    expect(tree).toMatchSnapshot();
  });

  it('should mount in a full DOM', () => {
    expect(mount(<Dropdown {...defaultProps} />).find('.dropdown').length).toBe(
      1
    );
  });

  it('should handle click on dropdown title', () => {
    const btn = tree.find('.dropdown__button');
    btn.simulate('click');
    expect(spyOnExpandDropdown).toHaveBeenCalled();
  });
});
