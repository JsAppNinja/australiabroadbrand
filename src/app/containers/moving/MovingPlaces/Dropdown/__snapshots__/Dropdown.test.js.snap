// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown tests should render dropdown section 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Dropdown
    data={
      Object {
        "item": Object {
          "mainAnswer": "You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.",
          "mainQuestion": "How do I move my Australia Broadband service with me?",
          "questions": Array [
            Object {
              "answer": "That’s great! Simply follow the steps above",
              "question": "My new address is nbn™ ready. What do I do?",
            },
            Object {
              "answer": "If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.",
              "question": "What if my new address is not nbn™ ready?",
            },
          ],
        },
      }
    }
    expandDropdown={[MockFunction]}
    openedDropdownIdx={null}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="dropdown__button"
          onClick={[Function]}
        >
          <p
            className="dropdown__button__main-question"
          >
            How do I move my Australia Broadband service with me?
          </p>
          <div
            className="dropdown__button__circle"
          >
            <Arrow
              rotate={true}
            />
          </div>
        </div>,
        <AnimateHeight
          animateOpacity={false}
          animationStateClasses={
            Object {
              "animating": "rah-animating",
              "animatingDown": "rah-animating--down",
              "animatingToHeightAuto": "rah-animating--to-height-auto",
              "animatingToHeightSpecific": "rah-animating--to-height-specific",
              "animatingToHeightZero": "rah-animating--to-height-zero",
              "animatingUp": "rah-animating--up",
              "static": "rah-static",
              "staticHeightAuto": "rah-static--height-auto",
              "staticHeightSpecific": "rah-static--height-specific",
              "staticHeightZero": "rah-static--height-zero",
            }
          }
          applyInlineTransitions={true}
          delay={0}
          duration={500}
          easing="ease"
          height="auto"
          style={Object {}}
        >
          <section
            className="dropdown__content"
          >
            <p>
              You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.
            </p>
            <React.Fragment>
              <p>
                My new address is nbn™ ready. What do I do?
              </p>
              <p>
                That’s great! Simply follow the steps above
              </p>
            </React.Fragment>
            <React.Fragment>
              <p>
                What if my new address is not nbn™ ready?
              </p>
              <p>
                If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
              </p>
            </React.Fragment>
          </section>
        </AnimateHeight>,
      ],
      "className": "dropdown",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p
              className="dropdown__button__main-question"
            >
              How do I move my Australia Broadband service with me?
            </p>,
            <div
              className="dropdown__button__circle"
            >
              <Arrow
                rotate={true}
              />
            </div>,
          ],
          "className": "dropdown__button",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "How do I move my Australia Broadband service with me?",
              "className": "dropdown__button__main-question",
            },
            "ref": null,
            "rendered": "How do I move my Australia Broadband service with me?",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Arrow
                rotate={true}
              />,
              "className": "dropdown__button__circle",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "rotate": true,
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "animateOpacity": false,
          "animationStateClasses": Object {
            "animating": "rah-animating",
            "animatingDown": "rah-animating--down",
            "animatingToHeightAuto": "rah-animating--to-height-auto",
            "animatingToHeightSpecific": "rah-animating--to-height-specific",
            "animatingToHeightZero": "rah-animating--to-height-zero",
            "animatingUp": "rah-animating--up",
            "static": "rah-static",
            "staticHeightAuto": "rah-static--height-auto",
            "staticHeightSpecific": "rah-static--height-specific",
            "staticHeightZero": "rah-static--height-zero",
          },
          "applyInlineTransitions": true,
          "children": <section
            className="dropdown__content"
          >
            <p>
              You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.
            </p>
            <React.Fragment>
              <p>
                My new address is nbn™ ready. What do I do?
              </p>
              <p>
                That’s great! Simply follow the steps above
              </p>
            </React.Fragment>
            <React.Fragment>
              <p>
                What if my new address is not nbn™ ready?
              </p>
              <p>
                If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
              </p>
            </React.Fragment>
          </section>,
          "delay": 0,
          "duration": 500,
          "easing": "ease",
          "height": "auto",
          "style": Object {},
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.
              </p>,
              Array [
                <React.Fragment>
                  <p>
                    My new address is nbn™ ready. What do I do?
                  </p>
                  <p>
                    That’s great! Simply follow the steps above
                  </p>
                </React.Fragment>,
                <React.Fragment>
                  <p>
                    What if my new address is not nbn™ ready?
                  </p>
                  <p>
                    If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
                  </p>
                </React.Fragment>,
              ],
            ],
            "className": "dropdown__content",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.",
              },
              "ref": null,
              "rendered": "You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": "0",
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <p>
                    My new address is nbn™ ready. What do I do?
                  </p>,
                  <p>
                    That’s great! Simply follow the steps above
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "My new address is nbn™ ready. What do I do?",
                  },
                  "ref": null,
                  "rendered": "My new address is nbn™ ready. What do I do?",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "That’s great! Simply follow the steps above",
                  },
                  "ref": null,
                  "rendered": "That’s great! Simply follow the steps above",
                  "type": "p",
                },
              ],
              "type": Symbol(react.fragment),
            },
            Object {
              "instance": null,
              "key": "1",
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  <p>
                    What if my new address is not nbn™ ready?
                  </p>,
                  <p>
                    If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
                  </p>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "What if my new address is not nbn™ ready?",
                  },
                  "ref": null,
                  "rendered": "What if my new address is not nbn™ ready?",
                  "type": "p",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.",
                  },
                  "ref": null,
                  "rendered": "If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.",
                  "type": "p",
                },
              ],
              "type": Symbol(react.fragment),
            },
          ],
          "type": "section",
        },
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="dropdown__button"
            onClick={[Function]}
          >
            <p
              className="dropdown__button__main-question"
            >
              How do I move my Australia Broadband service with me?
            </p>
            <div
              className="dropdown__button__circle"
            >
              <Arrow
                rotate={true}
              />
            </div>
          </div>,
          <AnimateHeight
            animateOpacity={false}
            animationStateClasses={
              Object {
                "animating": "rah-animating",
                "animatingDown": "rah-animating--down",
                "animatingToHeightAuto": "rah-animating--to-height-auto",
                "animatingToHeightSpecific": "rah-animating--to-height-specific",
                "animatingToHeightZero": "rah-animating--to-height-zero",
                "animatingUp": "rah-animating--up",
                "static": "rah-static",
                "staticHeightAuto": "rah-static--height-auto",
                "staticHeightSpecific": "rah-static--height-specific",
                "staticHeightZero": "rah-static--height-zero",
              }
            }
            applyInlineTransitions={true}
            delay={0}
            duration={500}
            easing="ease"
            height="auto"
            style={Object {}}
          >
            <section
              className="dropdown__content"
            >
              <p>
                You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.
              </p>
              <React.Fragment>
                <p>
                  My new address is nbn™ ready. What do I do?
                </p>
                <p>
                  That’s great! Simply follow the steps above
                </p>
              </React.Fragment>
              <React.Fragment>
                <p>
                  What if my new address is not nbn™ ready?
                </p>
                <p>
                  If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
                </p>
              </React.Fragment>
            </section>
          </AnimateHeight>,
        ],
        "className": "dropdown",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p
                className="dropdown__button__main-question"
              >
                How do I move my Australia Broadband service with me?
              </p>,
              <div
                className="dropdown__button__circle"
              >
                <Arrow
                  rotate={true}
                />
              </div>,
            ],
            "className": "dropdown__button",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "How do I move my Australia Broadband service with me?",
                "className": "dropdown__button__main-question",
              },
              "ref": null,
              "rendered": "How do I move my Australia Broadband service with me?",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Arrow
                  rotate={true}
                />,
                "className": "dropdown__button__circle",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "rotate": true,
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "animateOpacity": false,
            "animationStateClasses": Object {
              "animating": "rah-animating",
              "animatingDown": "rah-animating--down",
              "animatingToHeightAuto": "rah-animating--to-height-auto",
              "animatingToHeightSpecific": "rah-animating--to-height-specific",
              "animatingToHeightZero": "rah-animating--to-height-zero",
              "animatingUp": "rah-animating--up",
              "static": "rah-static",
              "staticHeightAuto": "rah-static--height-auto",
              "staticHeightSpecific": "rah-static--height-specific",
              "staticHeightZero": "rah-static--height-zero",
            },
            "applyInlineTransitions": true,
            "children": <section
              className="dropdown__content"
            >
              <p>
                You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.
              </p>
              <React.Fragment>
                <p>
                  My new address is nbn™ ready. What do I do?
                </p>
                <p>
                  That’s great! Simply follow the steps above
                </p>
              </React.Fragment>
              <React.Fragment>
                <p>
                  What if my new address is not nbn™ ready?
                </p>
                <p>
                  If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
                </p>
              </React.Fragment>
            </section>,
            "delay": 0,
            "duration": 500,
            "easing": "ease",
            "height": "auto",
            "style": Object {},
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <p>
                  You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.
                </p>,
                Array [
                  <React.Fragment>
                    <p>
                      My new address is nbn™ ready. What do I do?
                    </p>
                    <p>
                      That’s great! Simply follow the steps above
                    </p>
                  </React.Fragment>,
                  <React.Fragment>
                    <p>
                      What if my new address is not nbn™ ready?
                    </p>
                    <p>
                      If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
                    </p>
                  </React.Fragment>,
                ],
              ],
              "className": "dropdown__content",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.",
                },
                "ref": null,
                "rendered": "You’re moving; no worries! The first step is to check your address on our address checker and find out if your address is nbn™ ready.",
                "type": "p",
              },
              Object {
                "instance": null,
                "key": "0",
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    <p>
                      My new address is nbn™ ready. What do I do?
                    </p>,
                    <p>
                      That’s great! Simply follow the steps above
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "My new address is nbn™ ready. What do I do?",
                    },
                    "ref": null,
                    "rendered": "My new address is nbn™ ready. What do I do?",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "That’s great! Simply follow the steps above",
                    },
                    "ref": null,
                    "rendered": "That’s great! Simply follow the steps above",
                    "type": "p",
                  },
                ],
                "type": Symbol(react.fragment),
              },
              Object {
                "instance": null,
                "key": "1",
                "nodeType": "function",
                "props": Object {
                  "children": Array [
                    <p>
                      What if my new address is not nbn™ ready?
                    </p>,
                    <p>
                      If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "What if my new address is not nbn™ ready?",
                    },
                    "ref": null,
                    "rendered": "What if my new address is not nbn™ ready?",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.",
                    },
                    "ref": null,
                    "rendered": "If your new address is not nbn™ ready, unfortunately you can’t connect with Australia Broadband at your new address. Australia Broadband only provides internet services on the nbn™ and does not provide ADSL services. If you are an existing customer, all you need to do is email us at customercare@australiabroadband.net.au and let us know you are moving to an address not supported by the nbn™ and the date you would like us to disconnected your Australia Broadband nbn™ service from your old address. With Australia Broadband, there are no lock-in contracts and no early termination fees, so you can disconnect your nbn™ service at any time by giving us 1 months notice.",
                    "type": "p",
                  },
                ],
                "type": Symbol(react.fragment),
              },
            ],
            "type": "section",
          },
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
